cmd/tendermint/commands/reindex_event.go

```diff
@@ -141,7 +141,7 @@ func loadEventSinks(cfg *tmcfg.Config) ([]indexer.EventSink, error) {
 	return eventSinks, nil
 }
 
-func loadStateAndBlockStore(cfg *tmcfg.Config) (*store.BlockStore, state.Store, error) {
+func loadStateAndBlockStore(cfg *tmcfg.Config) (_ *store.BlockStore, _ state.Store, rerr error) {
```

Comment:
these `rerr` values are not set anywhere, which makes this change a noop. I think you need to declare this as `err` and then be careful to not re-declare it later.

Reply:
I know how the defer statement works, I'm saying there is no code (as in the example,) in these patches which modifies this line? Theoretically something could, but nothing does.